### ISSUE TITLE
Ability to select ffmpeg.probe executable

### ISSUE THREAD

**Issue by laurentalacoque** [kkroening/ffmpeg-python#122]
Hey,
Thanks for your work!

In #57 you added support for ffprobing files and that's cool ;)
Could you please add the same `cmd` argument to probe as in `_out.py::run()` function ?
This would make it possible to select alternate versions of `ffmpeg` programmatically.

I'm sending you a PR ASAP

**Comment by laurentalacoque** [comment]
PR #123 should solve the issue.

Another hack might work in the meantime: setting the PATH environment 
```
# To select ffmpeg executable at path/to/ffmpeg/bin use
FFMPEG_DIR = os.path.join("path","to","ffmpeg","bin")
os.environ['PATH'] = FFMPEG_DIR + ";" + os.environ['PATH'] # Windows uses ';' as path separator, use ":" on linux
```

**Comment by kkroening** [comment]
Merged #123. Thanks!
